src/utils.ts

```diff
@@ -63,26 +63,30 @@ ${templateCode}
   };
 };
 
-export const defaultOnSubmitCode = `"onSubmit": (submitEvent) => console.dir(submitEvent)`;
-
-export const defaultArgumentsCode = (apiKey: string) => `"apiKey": "${apiKey}",
```

Comment:
This is used [here](https://github.com/Constructor-io/constructorio-ui-autocomplete/blob/main/src/stories/Autocomplete/Hook/UserEvents.stories.tsx#L4) and [here](https://github.com/Constructor-io/constructorio-ui-autocomplete/blob/main/src/stories/Autocomplete/Component/UserEvents.stories.tsx#L4)  

I can remove them and update the args if you'd like to

Reply:
Good catch! Fixed here [13dab01](https://github.com/Constructor-io/constructorio-ui-autocomplete/pull/67/commits/13dab0165f37c53be0c4872598acb8de960a4343)